laura_vdearmas/js-hello-world/index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>LauraV JS</title>
+    
+</head>
+<body>
+    
+</body>
+<script src="alert.js"></script>
```

Comment:
```suggestion
<body>
    
  <script src="alert.js"></script>
</body>
```

Esto debe estar dentro de la etiqueta body y antes de q se cierre